tests/common/plugins/pdu_controller/__init__.py

```diff
@@ -18,6 +18,11 @@ def pdu_controller(duthosts, enum_rand_one_per_hwsku_hostname, conn_graph_facts,
     duthost = duthosts[enum_rand_one_per_hwsku_hostname]
     inv_mgr = duthost.host.options["inventory_manager"]
     pdu_host_list = inv_mgr.get_host(duthost.hostname).get_vars().get("pdu_host")
+    if not pdu_host_list:
+        logging.info("No 'pdu_host' is defined in inventory file for {}. Unable to create pdu_controller".format(
+            duthost.hostname))
+        yield None
+        return
```

Comment:
This is a good change but doesn't seem to belong to this PR? Can you separate this change in a different PR?

Reply:
@yxieca I pulled that changes and will create separate PR, please re-review